### PR TITLE
refactor: optimize UpdateCollectionsFiatValue command

### PR DESCRIPTION
**Pull request by shahin-hq**
## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] I added a storybook entry for the component that was added _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Comment by shahin-hq** [comment]
Noticed a mistake here, fixing..

**Comment by shahin-hq** [comment]
Wrong alarm, all good, can be reviewed
tl;dr 
the `User::updateCollectionsValue();` doesn't depend on the  `Collection::updateFiatValue()` call, so we can queue it as we wish.

**Review by oXtxNt9U — APPROVED** [review]

**Review by crnkovic — APPROVED** [review]
